### ISSUE TITLE
x:SetterTargetType on Animation is considered an error at design time

### ISSUE THREAD

**Issue by MihaMarkic** [ForNeVeR/AvaloniaRider#382]
For some reason x:SetterTargetType on Animation type is considered an error at design time. At runtime it works just fine. Plugin version 1.3.2, Rider 2024.1.4 and Avalonia 11.0.11.
To repro create a new Avalonia project and slap Animation into Window.Resources node:
```xaml
<Animation x:Key="FadeLoading"
           x:SetterTargetType="TextBlock">
</Animation>
```

**Comment by ForNeVeR** [comment]
Please explain what do you mean by "considered an error at design time"?

**Comment by MihaMarkic** [comment]
This 
![image](https://github.com/user-attachments/assets/09719720-3148-4a00-8ef2-83bc8fb01ba3)

**Comment by ForNeVeR** [comment]
Please report to Rider. The plugin has nothing to do with the code highlighting.

**Comment by MihaMarkic** [comment]
Ah, I thought it was plugin doing that. Thanks, will do.

**Comment by MihaMarkic** [comment]
For curious - here is the Rider [issue](https://youtrack.jetbrains.com/issue/RSRP-498037), consider voting for it.

**Comment by MihaMarkic** [comment]
After almost two years, the fix is slated for 2026.2! Fingers crossed.
